src/widgets/index.js

```diff
@@ -4,8 +4,9 @@
  */
 
 export { Button, ProgressBar } from 'react-native-ui-components';
-
+// eslint-disable-next-line import/no-cycle
```

Comment:
In the above files if we export directly from the files rather than from `.` we can remove the cycle, I believe

Reply:
I thought about doing that, but did this for consistency. Will revert back.

Reply:
Resolved with 7ae9c50